Assets/HoloToolkit/Sharing/Scripts/Utilities/SharingServiceStatus.cs

```diff
@@ -0,0 +1,61 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.UI;
+
+namespace HoloToolkit.Sharing
+{
+    public class SharingServiceStatus : MonoBehaviour
+    {
+        enum Status
+        {
+            Disconnected,
+            Connecting,
+            Connected,
+        }
+
+
+        [Tooltip("Reference to Text UI control where the Status info should be displayed.")]
+        public Text Text;
```

Comment:
@alexdrenea yeah I have certainly changed colors of the font on the TextMesh. 
Will share a screenshot with you soon as I get a chance.